lib/WorseReflection/Bridge/TolerantParser/Reflection/ReflectionProperty.php

```diff
@@ -65,7 +67,11 @@ public function declaringClass(): ReflectionClassLike
 
     public function name(): string
     {
-        return (string) $this->variable->getName();
+        if ($this->name) {
+            return $this->name;
+        }
+        $this->name = (string) $this->variable->getName();
```

Comment:
reading the name from the node is actually very expensive and should be avoided.